File: sandbox/node/src/FaxLineGetDefaultExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.FaxLineApi();

const number = "[FAX_NUMBER]";

apiCaller.faxLineGet(
    number,
).then(response => {
  console.log(response.body);
}).catch(error => {
  console.log("Exception when calling FaxLine#faxLineGet:");
  console.log(error.body);
});

File: sandbox/python/src/FaxLineGetDefaultExample.py
from datetime import date, datetime
from pprint import pprint

from dropbox_sign import ApiClient, ApiException, Configuration, api, models

configuration = Configuration()

with ApiClient(configuration) as api_client:
    try:
        response = api.FaxLineApi(api_client).fax_line_get(
            number="[FAX_NUMBER]",
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling FaxLine#fax_line_get: %s\n" % e)
